app/src/main/java/io/github/drp08/studypal/presentation/screens/DailyCalendarScreen.kt

```diff
@@ -25,39 +28,62 @@ object DailyCalendarScreen : Screen {
 
     @Composable
     override fun Content() {
-            val viewModel = DailyCalendarView()
-            val currentDate by viewModel.currentDate.collectAsState(initial = Clock.System.now().toLocalDateTime(TimeZone.currentSystemDefault()).date)
-            val events by viewModel.events.collectAsState(initial = emptyList())
+//            val viewModel = DailyCalendarView()
+//            val currentDate by viewModel.currentDate.collectAsState(initial = Clock.System.now().toLocalDateTime(TimeZone.currentSystemDefault()).date)
+//            val events by viewModel.events.collectAsState(initial = emptyList())
+        var currentView by remember { mutableStateOf(CalendarView.DAILY) }
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(16.dp)
+        ) {
+            CalendarViewSwitcher(
+                currentView = currentView,
+                onViewChange = { newView -> currentView = newView })
+            Spacer(modifier = Modifier.height(16.dp))
+
+            when (currentView) {
+                CalendarView.DAILY -> DailyView()
+                CalendarView.WEEKLY -> WeeklyCalendarScreen.Content()
+                CalendarView.MONTHLY -> MonthlyCalendarScreen.Content()
+            }
+        }
+    }
+    }
 
-            Column(
-                modifier = Modifier
-                    .fillMaxSize()
-                    .padding(16.dp)
+@Composable
+fun DailyView() {
+    val viewModel = DailyCalendarView()
+    val currentDate by viewModel.currentDate.collectAsState(initial = Clock.System.now().toLocalDateTime(TimeZone.currentSystemDefault()).date)
```

Comment:
We are using epochMillis time everywhere, does that comply here or do you need to change it? You could perhaps wait for #8 to be merged before